docs/tutorials/python/tutorial_scripts/evaluation.py

```diff
@@ -0,0 +1,66 @@
+"""
+Here is where you'll find the code for the Evaluation tutorial.
+"""
+
+from synapseclient import Synapse
+from synapseclient.models import Evaluation, Project
+
+syn = Synapse()
+
+syn.login()
+
+# Retrieve the Project where your Evaluation will be stored
+project = Project(name="test_project_b6a2a3c0-ed60-4127-b34f-414a9a0d8ba1").get()
+project_id = project.id
+
+print(f"Working within Project: {project_id}")
+
+# Create a new Evaluation object
+evaluation = Evaluation(
+    name="My Challenge Evaluation - Round 1",
+    description="Evaluation for my data challenge",
+    content_source=project_id,
+    submission_instructions_message="Submit CSV files only",
+    submission_receipt_message="Thank you for your submission!",
+)
+
+# Create the Evaluation on Synapse
+evaluation.store()
+
+print("Evaluation has been created with the following name and description:")
+print(evaluation.name)
+print(evaluation.description)
+
+# Update the Evaluation object's name and description
+evaluation.name = "My Challenge Evaluation - Round 2"
+evaluation.description = "Updated description for my evaluation"
+
+# Update the Evaluation on Synapse
+evaluation.store()
+
+print("Evaluation has been updated with the following name and description:")
+print(evaluation.name)
+print(evaluation.description)
+
+# Confirm what's in Synapse matches the evaluation stored
+from_synapse = Evaluation(id=evaluation.id).get()
+
+print("The following evaluation has been retrieved from Synapse:")
+print(from_synapse)
+
+# Get the Evaluation's ACL
+acl = evaluation.get_acl()
+
+# Add a new entry to the ACL's 'resourceAccess' list
+acl["resourceAccess"].append({"principalId": 3453016, "accessType": ["READ", "SUBMIT"]})
```

Comment:
Why are we hard-coding the principal ARN here? Can it can be any principal ID?

Reply:
Good point! Yes, it can be any principal that currently exists on Synapse. I've changed this to programmatically feed a principal ID provided by the user.